lumicks/pylake/fdcurve.py

```diff
@@ -199,3 +200,9 @@ def range_selector(self, show=True):
 
     def distance_range_selector(self, show=True, max_gap=0):
         return FdDistanceRangeSelectorWidget(self, show=show, max_gap=max_gap)
+
+
+@deprecated.deprecated("The class FDCurve was renamed to FdCurve.")
+class FDCurve(FdCurve):
+    def __init__(self, *args, **kwargs):
+        FdCurve.__init__(*args, **kwargs)
```

Comment:
1. Include the version as well.
2. Do you need `__init__`?
```suggestion
@deprecated.deprecated("The class FDCurve was renamed to FdCurve.")
class FDCurve(FdCurve):
    pass
```

Reply:
1. Added.
2. Nope we don't. Good point. Fixed! 👍